components/renku_data_services/migrations/versions/ce1a111d6694_authz_add_intial_schema.py

```diff
@@ -23,6 +24,15 @@
 def upgrade() -> None:
     config = AuthzConfig.from_env()
     client = config.authz_client()
+
+    try:
+        existing_schema = client.ReadSchema(ReadSchemaRequest())
+        if "definition user" in existing_schema.schema_text:
+            logging.info(f"Skipping {revision}, as Authz is likely already populated.")
+            return
+    except RpcError:
+        pass
```

Comment:
Is this to catch the issue you mention in the PR description?

The problem is that this will also catch other issues that are unrelated but potentially important.

The proper way to address the problem in the PR description is to delete all relationships that reference user_namespace.

This can hide errors that are important.

Reply:
I you would like, we can delay this until the migrations have a proper fix. I do not know how upgrade/downgrades of the authzed schema need to be handled.